### DISCUSSION TITLE
Default value for `commitMessageExtra`

### DISCUSSION THREAD

**Discussion by ahatzz11** [renovatebot/renovate#16212] [Request Help] [answered]
👋 I am looking at the docs for `commitMessageExtra` and I'm having a hard time tracking down the default value: https://docs.renovatebot.com/configuration-options/#commitmessageextra

The docs just say `[template]`. I've dug through the code a little bit and I think that it is this, but I'm not sure if I'm right here:
https://github.com/renovatebot/renovate/blob/03b0d2a24e72b78e77a2b0e2e383bfa8b86e1251/lib/workers/repository/updates/generate.spec.ts#L136-L137

Could someone help me confirm the default value for this field? We are looking at tweaking it to include `from v{{currentVersion}}` and I want to ensure I understand how the defaults work properly before messing with things too much. Thanks!

**Comment by rarkins** [comment, marked as answer]
Its default value is here: https://github.com/renovatebot/renovate/blob/29a272c78fb2b9a31be24a44be63691b27ab654b/lib/config/options/index.ts#L1622-L1630

**Reply by ahatzz11** [reply]
Sweet, thanks for the link! That helps lots.

**Comment by HonkingGoose** [comment]
The docs displaying `[template]` is a bug, we're working on fixing it:

- https://github.com/renovatebot/renovate/issues/16216

**Reply by ahatzz11** [reply]
That's awesome, thanks!
